File: app/src/main/java/com/example/amit/uniconnexample/Chatstartmodel.java
package com.example.amit.uniconnexample;

/**
 * Created by amit on 9/12/16.
 */

public class Chatstartmodel {

private String msg1,msg2,time1,time2;
    public Chatstartmodel(){

    }
    public Chatstartmodel(String msg1,String time1){
        this.msg1=msg1;
        this.time1=time1;
    }

    public Chatstartmodel(String msg1, String msg2, String time1, String time2) {
        this.msg1 = msg1;
        this.msg2 = msg2;
        this.time1 = time1;
        this.time2 = time2;
    }

    public void setMsg1(String msg1){
        this.msg1=msg1;
    }
    public void setMsg2(String msg2){
        this.msg2=msg2;
    }
    public String getMsg1(){
        return msg1;
    }
    public String getMsg2(){
        return msg2;
    }

    public String getTime2() {
        return time2;
    }

    public String getTime1() {
        return time1;
    }
}

File: app/src/main/java/com/example/amit/uniconnexample/Model/Chatstartmodel.kt
package com.example.amit.uniconnexample

/**
 * Created by amit on 9/12/16.
 */

class Chatstartmodel {

    var msg1: String? = null
    var msg2: String? = null
    var time1: String? = null
        private set
    val time2: String

    constructor() {

    }

    constructor(msg1: String, time1: String) {
        this.msg1 = msg1
        this.time1 = time1
    }

    constructor(msg1: String, msg2: String, time1: String, time2: String) {
        this.msg1 = msg1
        this.msg2 = msg2
        this.time1 = time1
        this.time2 = time2
    }
}
